pom.xml

```diff
@@ -22,25 +22,9 @@
         </license>
     </licenses>
     <scm>
-        <connection>scm:git:git://github.com/smoope/j2objc-maven-plugin.git</connection>
-        <url>https://github.com/smoope/j2objc-maven-plugin</url>
+        <connection>scm:git:git://github.com/kenwdelong/j2objc-maven-plugin.git</connection>
```

Comment:
I'd like to not change this section since master repo is still in scm:git:git://github.com/smoope/j2objc-maven-plugin.git